src/hardware/mouse/mouseif_dos_driver.cpp

```diff
@@ -238,6 +232,11 @@ static uint16_t get_pos_y()
 	return static_cast<uint16_t>(std::lround(pos_y)) & state.granularity_y;
 }
 
+static uint16_t mickey_counter_to_reg16(const float x)
+{
+	return static_cast<uint16_t>(std::lround(x));
+}
```

Comment:
How are `x` values outside the uint16_t range handled?

Reply:
The `update_mickey` lambda should guarantee it is in range.